eventhandlers/main.py

```diff
@@ -21,4 +25,10 @@
     logger.info(f'Finished Processing Messages')
 except Exception as e:
     data = traceback.format_exc()
-    logger.error(f'Error Processing event: {data}')
\ No newline at end of file
+    logger.error(f'Error Processing event: {data}')
+
+def exit_handler():
```

Comment:
This is a great idea! I think there's a lot of extra checks we can add during shutdown to avoid having dangling threads or actions in future versions

Reply:
Yeah, I think that will be a good Idea